packages/ckeditor5-basic-styles/src/bold/boldui.ts

```diff
@@ -21,20 +23,20 @@ export default class BoldUI extends Plugin {
 	/**
 	 * @inheritDoc
 	 */
-	static get pluginName() {
+	public static get pluginName(): 'BoldUI' {
 		return 'BoldUI';
 	}
 
 	/**
 	 * @inheritDoc
 	 */
-	init() {
+	public init(): void {
 		const editor = this.editor;
 		const t = editor.t;
 
 		// Add bold button to feature components.
 		editor.ui.componentFactory.add( BOLD, locale => {
-			const command = editor.commands.get( BOLD );
+			const command: AttributeCommand = editor.commands.get( BOLD )!;
```

Comment:
This is most probably not needed because of `CommandsMap` (but it's not a mistake, this is also ok).